src/components/Context.tsx

```diff
@@ -1,18 +1,31 @@
-import * as React from 'react'
-import { SqVerificationDetails, SqError, SqVerificationResult } from './models'
+import * as React from 'react';
+import { SqVerificationDetails, SqError, SqVerificationResult } from './models';
 
 export interface ContextInterface {
-  formId?: string
-  applePayState?: string
-  googlePayState?: string
-  masterpassState?: string
-  onCreateNonce?: (event: React.MouseEvent) => {}
+  /** Unique form ID */
+  formId?: string;
+  /** Apple pay state (loading, unavailable, ready) */
+  applePayState?: string;
```

Comment:
Could we do `applePayState?: 'loading' | 'unavailable' | 'ready';` for these?

Reply:
Done!